build.psm1

```diff
@@ -2019,6 +2019,9 @@ function Find-Dotnet() {
 
     if (-not (precheck 'dotnet' "Still could not find 'dotnet', restoring PATH.")) {
         $env:PATH = $originalPath
+    } else {
+        $dotnetInfo = (dotnet --info)
+        Write-Log "dotnet --info: `n$dotnetInfo"
```

Comment:
`dontet --info` would be too verbose I think. How about just use "dotnet --version"?

Reply:
Really this is needed only for moving to new .Net Core versions. I think we could remove the logging at all. `dotnet --version` is not so useful because don't show versions of all components.

Reply:
Removing sounds good to me.